lib/gds_api/test_helpers/local_links_manager.rb

```diff
@@ -5,13 +5,14 @@ module TestHelpers
     module LocalLinksManager
       LOCAL_LINKS_MANAGER_ENDPOINT = Plek.current.find("local-links-manager")
 
-      def stub_local_links_manager_has_a_link(authority_slug:, lgsl:, lgil:, url:)
+      def stub_local_links_manager_has_a_link(authority_slug:, lgsl:, lgil:, url:, country_name:)
```

Comment:
As these are helpers and used in other codebases I'd add a default value for country name. Otherwise it'll be a breaking change.
```suggestion
      def stub_local_links_manager_has_a_link(authority_slug:, lgsl:, lgil:, url:, country_name: "Wales")
```

Reply:
Good spot - thanks Sean - added `England` as the default.